src/providers/Votes.js

```diff
@@ -0,0 +1,34 @@
+import React, { useContext, useMemo } from 'react'
+import PropTypes from 'prop-types'
+import { useDisputableVotes } from '../hooks/useDisputableVotes'
+
+const VotesContext = React.createContext({
+  votes: [],
+  loading: true,
+})
+
+function VotesProvider({ children }) {
```

Comment:
We may end up not need these contexts in the future. I was thinking we could just open a subscription when on a route, and unsub when leaving. I'm a little concerned about down the wire data costs of the polling if we leave everything open, especially on mobile. I think for now this is fine because we're not polling on these yet.

I've added them here because getting the votes at the moment takes a bit of time due to all of the `describeScript` calls, and bouncing between `proposals` and `proposalsSingle` without it repeats all of that work, making it quite irritating from a UX perspective.